frontend/packages/ceph-storage-plugin/integration-tests/tests/3-tests/multiple-storage-class-selection.scenario.ts

```diff
@@ -0,0 +1,119 @@
+import { ExpectedConditions as until, browser } from 'protractor';
+import { execSync, ExecFileOptionsWithStringEncoding } from 'child_process';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { click } from '@console/shared/src/test-utils/utils';
+import { DeviceSet } from '../../../src/components/ocs-install/ocs-request-data';
+import { OCS_INTERNAL_CR_NAME } from '../../../src/constants';
+import { getCurrentDeviceSetIndex } from '../../../src/utils/add-capacity';
+import { mockSC, mockNoProvisionerSC } from '../../mocks/storage-class';
+import {
+  goToInstalledOperators,
+  currentACSelector,
+  clickKebabAction,
+} from '../../views/add-capacity.view';
+import { NS } from '../../utils/consts';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+
+const fetchStorageClusterYAML = () => {
+  const options: ExecFileOptionsWithStringEncoding = {
+    encoding: 'utf8',
+  };
```

Comment:
Not required default is utf8

Reply:
Default is 'Buffer'. https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_optio

Reply:
Ack